### ISSUE TITLE
Question about generated videos?

### ISSUE THREAD

**Issue by Fitzwong** [lucidrains/nuwa-pytorch#8]
There are a lot of negative numbers and very small decimals (like 5e-1). But the loss degrades normally when training.
Is that a normal situation? How can I make the result visible?
